doc/field_guide__yso.md

```diff
@@ -0,0 +1,76 @@
+## Young Stellar Objects (yso)
+
+Young stellar objects (YSOs) exhibit a wide range of variability types,
+with many subclasses depending on the dominant physics in the system.
+
+
+Some of the lightcurve possibilities for YSOs include:
+
+
+
+

```

Comment:
```suggestion
```

Reply:
?

Reply:
Since the linting works only on modified files, I did this file by hand (when I had the malloc errors I did some python plus venv switching, and for some reason this file didn't throw lint errors on my terminal, but did when committed). The blank lines seemed to have escaped.